src/pawelak-arek/Fsd.Pawelak.Cs/Fsd.Arek.Cs2/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Fsd.Arek.Cs2.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Arek.Cs2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ConsoleMenu FactoryAnimalsProg = new ConsoleMenu();
```

Comment:
FactoryAnimalsProg to nazwa instancji czyli prywatna zmienna w funkcji. z małej litery i bez skrótów